daemon/algod/api/server/v2/utils.go

```diff
@@ -391,15 +390,15 @@ func convertSimulationResult(result simulation.Result) PreEncodedSimulateRespons
 	var evalOverrides *model.SimulationEvalOverrides
 	if result.EvalOverrides != (simulation.ResultEvalOverrides{}) {
 		evalOverrides = &model.SimulationEvalOverrides{
-			MaxLogSize:  result.EvalOverrides.MaxLogSize,
-			MaxLogCalls: result.EvalOverrides.MaxLogCalls,
+			SignaturesOptional: trueOrNil(result.EvalOverrides.SignaturesOptional),
```

Comment:
At one point I had thought that our encoder _would_ include values if they were non-nil pointers to zero values. But after local tests it seems that's not the case and that changing this line to `SignaturesOptional: &result.EvalOverrides.SignaturesOptional` would have the same effect.

Since this is surprising to me, I'd rather leave this code as-is and investigate more whether what I described was ever true, or I'm misremembering something